sprint/switch_branch.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# This script allows switching drupal branch, for example, from 8.9.x to 9.0.x
+# or back
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+if [[ $# != 1 ]]; then
+  echo "Please provide a branch to switch to. For example 'switch_branch.sh 9.0.x'"
+  exit 1
+fi
+
+
+target_branch=$1
+
+pushd drupal8
+set -x
+ddev exec "git stash && git reset --hard && git checkout ${target_branch} && git pull && git stash pop"
```

Comment:
Should we warn before destroying any changes?

Reply:
As above, it's doing it's level best not to destroy changes. And if it fails, the stash is still there. Not sure the level of effort that should be applied to this.

Reply:
The thing that worries me the most is not losing changes, but introducing the change to composer.json and composer.lock, and of course that getting into a patch.